src/main/java/fi/aalto/cs/apluscourses/intellij/utils/VfsUtil.java

```diff
@@ -4,15 +4,66 @@
 import com.intellij.openapi.vfs.VirtualFile;
 import com.intellij.openapi.vfs.VirtualFileVisitor;
 import java.nio.file.Path;
+import java.nio.file.Paths;
 import org.jetbrains.annotations.CalledWithReadLock;
 import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
 
 public class VfsUtil {
```

Comment:
missed this previously, but is there some special reason why we're shadowing the name of the JB's API classes?

Reply:
probably just too lazy to come up with an original name :laughing: